join.spec.md

```diff
@@ -187,9 +181,9 @@ Join a field with a particular subgraph.
 
```

Comment:
Previous line: this uses a term that isn't really defined. Probably should be more like "Declares that a field should be resolved within a particular subgraph" or something.

Reply:
Fixed